client/.env.sample

```diff
@@ -0,0 +1,9 @@
+API_URL=http://10.0.2.2:3000
+KEYCLOAK_URL=http://keycloak-server-snapscreen.74.207.224.48.xip.io/auth
+KEYCLOAK_REALM=SnapScreen
+KEYCLOAK_CLIENT_ID=android
+KEYCLOAK_REDIRECT_URI=keycloak-demo://app
+UNIFIED_PUSH_VARIANT_ID=3e2c6bd7-cd69-49eb-bd9e-a9360e79f1a9
+UNIFIED_PUSH_SECRET=15448edc-8896-4163-aca6-cbf783f9165b
```

Comment:
Right.  This is a sample file.  When we regenerate the values this file will be obsolete for anything but a reference.  The README instructions will include notes for how to copy this file to a .env file and where to find the values to replace.